.github/workflows/docker.yml

```diff
@@ -37,21 +37,22 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Build the Stack
-        run:  |
+        run: |
           cd src
           docker-compose -f local.yml build
 
       - name: Run DB Migrations
-        run:  |
+        run: |
           cd src
+          docker-compose -f local.yml run --rm django python manage.py makemigrations --merge --noinput
```

Comment:
When we merged and the migrations were out of order but had technically also been applied already it was getting upset. Since it's run once do you think it'll have fixed the issue and we can remove it now?

Reply:
It should just be building a fresh db every time so I'm not sure why it would have been a problem.

Reply:
perhaps there's another reason? I'm not really sure but [this](https://github.com/UCL/frrant/actions/runs/8452099681/job/23151772385) was the error output so maybe it caches the build within PRs (if that would explain the behaviour)

Reply:
Pretty sure that error means there's two migration files with the same earlier migration file listed as a dependency. Now that that's fixed I think it's best to remove it as it could mask any future problems.